server/game/cards/07-GR/SkreelSwarm.js

```diff
@@ -0,0 +1,28 @@
+const Card = require('../../Card.js');
+
+class SkreelSwarm extends Card {
+    // Play: Deal 1D to each enemy creature.
+    //
+    // Scrap: You may archive Skreel Swarm from your discard pile.
+    setupCardAbilities(ability) {
+        this.play({
+            effect: 'deal 1 damage to each enemy creature',
+            gameAction: ability.actions.dealDamage((context) => ({
+                amount: 1,
+                target: context.game.creaturesInPlay.filter(
+                    (card) => card.controller !== context.player
+                )
+            }))
+        });
+
+        this.scrap({
+            gameAction: ability.actions.archive((context) => ({
```

Comment:
it is an optional ability, is says "you may", right?

Reply:
Oops, great catch.  Done.